lib/user.rb

```diff
@@ -0,0 +1,30 @@
+require_relative "recipient"
+
+module Slack
+  class User < Recipient
+    attr_reader :slack_id, :name, :real_name
+
+    def initialize(slack_id, name, real_name)
+      super(slack_id, name)
+      @real_name = real_name
+    end
+
+    def self.get
+      url = "https://slack.com/api/users.list"
+      params = {
+        token: ENV["KEY"],
```

Comment:
In general don't name your environmental variable `"KEY"`, make the name a bit more specific.  

Also both the token and the URL would make great constants.